CovertInfiltration/Localization/XComGame.int

```diff
@@ -146,51 +146,51 @@ ActionPostNarrative=""
 
 [InfiltrationSize1 X2SoldierUnlockTemplate]
 DisplayName="Infiltration Size I"
-Summary="Squads of 5 soldiers no longer suffer any penalty to infiltration time."
+Summary="Squads of 5 soldiers no longer suffer any penalty to infiltration time and can reach 200% infiltration."
 
 [InfiltrationSize2 X2SoldierUnlockTemplate]
 DisplayName="Infiltration Size II"
-Summary="Squads of 6 soldiers no longer suffer any penalty to infiltration time."
+Summary="Squads of 6 soldiers no longer suffer any penalty to infiltration time and can reach 200% infiltration."
 
 [UpdatedFirewalls X2SitRepTemplate]
 FriendlyName="Updated Firewalls"
 Description="Enemy tech defense is increased by <Ability:UPDATED_FIREWALLS_HACK_DEFENSE_BONUS/> and towers cannot be hacked"
 
 [MentalReadiness X2SitRepTemplate]
 FriendlyName="Mental Readiness"
-Description="+<Ability:MENTAL_READINESS_VALUE/> crit, dodge, hack and defense against psi"
+Description="Squad gains +<Ability:MENTAL_READINESS_VALUE/> crit, dodge, hack and will"
 
 [LightningStrike X2SitRepTemplate]
 FriendlyName="Lightning Strike"
-Description="Units gain a mobility boost for the first few turns of battle while the squad remains concealed."
+Description="XCOM units gain a mobility boost for the first few turns of battle while the squad remains concealed."
```

Comment:
I dislike "first few turns". Needs to be an actual value (`<Ability:LIGHTNINGSTRIKEDURATION/>`)

Reply:
The problem is that this tag returns the duration minus one, because it's meant to be used on the effect tooltip and not count the current turn. Having it read "1+1" looks super weird if I put "+1" at the end.

Reply:
Just make a custom tag then